integration_tests/dbt_date/dbt_project.yml

```diff
@@ -0,0 +1,47 @@
+name: "trino_utils_dbt_date_integration_tests"
+version: "1.0"
+
+profile: "integration_tests"
+
+config-version: 2
+
+model-paths: ["models"]
+test-paths: ["tests"]
+seed-paths: ["data"]
+macro-paths: ["macros"]
+
+target-path: "target"
+clean-targets: ["target", "dbt_modules", "dbt_packages"]
+
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['trino_utils_dbt_utils_integration_tests', 'trino_utils', 'dbt_utils']
+  - macro_namespace: dbt_date
+    search_order: ['trino_utils_dbt_date_integration_tests', 'trino_utils', 'dbt_date']
+
+vars:
+    "dbt_date:time_zone": "UTC"
+
+quoting:
+    database: false
+    identifier: false
+    schema: false
+
+models:
+  dbt_date_integration_tests:
+    schema: dbt_date_integration_tests
+    materialized: table
+
+    # TODO: Enable tests while adding Trino shims
+    dates:
+      +enabled: false
+    dim_date:
+      +enabled: false
+    dim_date_fiscal:
+      +enabled: false
+    dim_hour:
+      +enabled: false
+    dim_week:
+      +enabled: true
+    test_dates:
+      +enabled: false
```

Comment:
A lot of tests are disabled. Is this on purpose?

Reply:
Yes, I focused on setting up a testing environment for trino shims of dbt-date. So for example, `trino__convert_timezone` macro added by RobbertDM is now tested. However, in order to enable all tests, trino shims for most of dbt-date macros would need to be created. Creating shims for all of them is quite a lot of work, so I think we could make a separate issue for that as a follow up task. In the meantime, if someone creates a shim for a specific macro they need, they would only need to enable the test that checks that one macro.